packages/fe-container/src/styles/_colors.less

```diff
@@ -8,4 +8,5 @@
 :root {
   --microlc-primary-color: #1890FF;
   --microlc-menu-entry-selected-bg-color: #E6F3FF;
```

Comment:
Since it is used not only for the menu, we should change the name of this variable to something general, like `--microlc-primary-color-tint-89`. Change also the entry in `constants.ts` accordingly.

Reply:
If you agree, I suggest to do this in another branch, in order to avoid conflicts with master

Reply:
Ok